### ISSUE TITLE
[Linux] clDNN doesn't work from the build 7483033 on Linux

### ISSUE THREAD

**Issue by Christywl** [intel/webml-polyfill#473]
**Test Env:**
Chromium Version: nightly build 70.0.3503.0 (**95ba69b**)
Platform: Linux Ubuntu 16.04(Dell XPS 13)[CPU: intel i7-8550U, GPU: Intel UHD Craphics 620(Kabylake GT2), Memory: 16GB]

**Expected Result:**
clDNN works on Linux.

**Actual Result:**
WebML clDNN doesn't work from the build https://github.com/otcshare/chromium-src/commit/7483033ae6f79859969dde28c3666374636e7788 on Linux:

 
```
$lspci
00:00.0 Host bridge: Intel Corporation Device 5914 (rev 08)
00:02.0 VGA compatible controller: Intel Corporation Device 5917 (rev 07)
00:04.0 Signal processing controller: Intel Corporation Skylake Processor Thermal Subsystem (rev 08)
00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21)
00:14.2 Signal processing controller: Intel Corporation Sunrise Point-LP Thermal subsystem (rev 21)
00:15.0 Signal processing controller: Intel Corporation Sunrise Point-LP Serial IO I2C Controller (rev 21)
00:15.1 Signal processing controller: Intel Corporation Sunrise Point-LP Serial IO I2C Controller (rev 21)
00:16.0 Communication controller: Intel Corporation Sunrise Point-LP CSME HECI (rev 21)
00:1c.0 PCI bridge: Intel Corporation Device 9d10 (rev f1)
00:1c.4 PCI bridge: Intel Corporation Sunrise Point-LP PCI Express Root Port (rev f1)
00:1c.5 PCI bridge: Intel Corporation Sunrise Point-LP PCI Express Root Port (rev f1)
00:1d.0 PCI bridge: Intel Corporation Device 9d18 (rev f1)
00:1f.0 ISA bridge: Intel Corporation Device 9d4e (rev 21)
00:1f.2 Memory controller: Intel Corporation Sunrise Point-LP PMC (rev 21)
00:1f.3 Audio device: Intel Corporation Device 9d71 (rev 21)
00:1f.4 SMBus: Intel Corporation Sunrise Point-LP SMBus (rev 21)
3a:00.0 Network controller: Qualcomm Atheros QCA6174 802.11ac Wireless Network Adapter (rev 32)
3b:00.0 Unassigned class [ff00]: Realtek Semiconductor Co., Ltd. RTS525A PCI Express Card Reader (rev 01)
3c:00.0 Non-Volatile memory controller: Toshiba America Info Systems Device 0116
```
**How to Reproduce:**
1. Install the chromium and launch
2. Visit https://brucedai.github.io/nt/examples/image_classification/index.html
3. Check WebML backend

Note: It works on my Desktop[CPU:i7-7700 CPU, GPU:Intel® HD Graphics 630 (Kaby Lake GT2)]:
```
$ lspci
00:00.0 Host bridge: Intel Corporation Device 591f (rev 05)
00:02.0 VGA compatible controller: Intel Corporation Device 5912 (rev 04)
```

**Comment by huningxin** [comment]
> Platform: Windows 10 (Dell XPS 13)

Is it a typo? What's the Linux version?

**Comment by Christywl** [comment]
Sorry, should be Linux 16.04(Dell XPS 13)

**Comment by Christywl** [comment]
This issue was fixed on the nightly build 3f6312d.
